docs/website/contents/for-developers/Glossary.md

```diff
@@ -0,0 +1,490 @@
+# Glossary
+
+## Active Slot Coefficient
+
+The active slot coefficient represents the chance a given slot elects one of the registered stake pools.
+In other words, the probability that each slot has any leaders.
+It is usually denoted as $f$.
+
+On the Cardano mainnet we have:
+
+$$f = 1/20$$
+
+So this means a 5% chance of electing one of the registered stakepools. Thus, under ideal circumstances, each slot has a 19/20 probability of having no leaders whatsoever.
+
+In the implementation we assume $f$ won't change. Dealing with changing $f$ is a hard problem which requires proper research.
+
+## Adversarial node
+
+A node that exhibits any behavior beyond the honest behaviors. EG it doesn't mint when it's elected. EG it mints multiple blocks when elected. EG it mints its block "too" late. EG it mints its block "too" soon. EG it mints invalid blocks. EG it sends headers but withholds the block, etc, etc, etc.
+
+## Adversary
+
+Security analyses typically assume the worst-case, ie the behavior of all adversarial nodes as well as any environmental interference is concerted (eg the adversasy "controls" more than just nodes, eg natural arising network delays).
+
+## Anchor
+
+A [chain fragment](#Prefix/tip/etc-and-fragment) always has an anchor [point](#point), which is the predecessor of the oldest [header](#header)/[block](#block) on the fragment.
```

Comment:
Minor suggestion: If every wiki-ish links like the ones you've added is targeting this same document, then my intuition is that we don't need them.

- They'll be a burden to maintain if we rename things, etc.
- They make the Markdown itself less legible.
- If someone is reading a description on this page and they see a word they don't recognize/etc, they should be able to CTRL-F to find its definition. To make this easier, we could add a symbol that's unique to the definition of the word, such as `## defn:Block` and so on. So if you want to find the definition of a word foo, you CTRL-F for `defn:foo`. (A very tame JavaScript one-liner can automate this: `document.querySelectorAll("h2").forEach(function (e) { e.innerHTML = "defn:" + e.innerHTML.trim(); })` eg at https://jsfiddle.net/c8ustjzh/.) Or maybe some cheeky symbol, such as ‡ instead of `defin:`?

That'd be much less noisy, easier to maintain, and only _slightly_ less convenient to use.

Reply:
> They'll be a burden to maintain if we rename things, etc.

I think Docusaurus check for broken links. We still have to fix these, but I don't think we'll change headings such a `## Slots` very often.

Reply:
> they should be able to CTRL-F to find its definition

This might be a problem. For instance, consider the word "slot" or even the mention of `k` or `scg`. It might take ~1 minute to hit `CTRL-F` till they find the right definition. So we need at least at `defn:foo` in the headers, as you suggested.

> Or maybe some cheeky symbol, such as ‡

I think it's better if we use something people can easily type :D

Reply:
```
## :block
## :slot
## :header
```

would probably suffice IMO and not be too much visual noise.

> I think it's better if we use something people can easily type :D

Good point! It will at least essentially always be on the screen and so possible to copy onto the clipboard 🤷

Reply:
I suggest: we use simply the colons.

Bonus script at the bottom of the file: `document.querySelectorAll("h2").forEach(function (e) { e.innerHTML = e.innerHTML.replace(':','<span style="color: rgba(0, 0, 0, 0.5);">:</span>';) })`